### DISCUSSION TITLE
Add status bar to Main window in Prism WPF generated template

### DISCUSSION THREAD

**Discussion by avazon** [microsoft/TemplateStudio#4661] [Q&A] [answered]
I would like to add a statusbar at the bottom of my main window in a WPF WTS generated Prism application. I think it would go into the shellwindow.xaml but I do not quite understand the code generated. I was hoping for a Grid where I could add a new Row at the end with a height of around 30px

**Comment by mrlacey** [comment, marked as answer]
You can easily add such a grid.

Assuming you're using a HamburgerMenu, you can put it inside the content

```
            <controls:HamburgerMenu.Content>
                <Grid>
                    <Grid.RowDefinitions>
                        <RowDefinition Height="*" />
                        <RowDefinition Height="30" />
                    </Grid.RowDefinitions>
                    <ContentControl x:Name="hamburgerMenuContentControl" />
                    <TextBlock Grid.Row="1" Background="BlueViolet" Text="MY STATUS BAR" />
                </Grid>
            </controls:HamburgerMenu.Content>
```

**Reply by avazon** [reply]
Thank You Matt!
